docs/tables/snowflake_session.md

```diff
@@ -0,0 +1,52 @@
+# Table: snowflake_session
+
+This Account Usage view provides information on the session, including information on the authentication method to Snowflake and the Snowflake login event. Snowflake returns one row for each session created over the last year.
+
+**Note**:
+
+- This table requires a `Snowflake warehouse` to query. You can set it by `warehouse` config argument in Steampipe connection config.
+- Latency for the view may be up to 180 minutes (3 hours).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  session_id,
+  user_name,
+  authentication_method,
+  created_on,
+  client_environment ->> 'APPLICATION' as clint_application
+from
+  snowflake_session;
+```
+
+### List distinct authentication methods used in last year
+
+```sql
+select distinct
+  user_name,
+  authentication_method
+from
+  snowflake_session
+order by
+  user_name;
+```
+
+### List sessions authencticated without Snowflake MFA with passsword in 30 days
```

Comment:
```suggestion
### List sessions authenticated without Snowflake MFA with password in last 30 days
```

Please be careful of typos (multiple in this title), use a spellchecker if required